website/docs/tasks/list-accounts.md

```diff
@@ -0,0 +1,44 @@
+---
+title: list accounts
+---
+
+## Overview
+
+Taq `list accounts` is a task exposed by the [Flextesa plugin](/docs/plugins/plugin-basics) which lists the accounts and balances that exist 
```

Comment:
Link to the flextesa plugin here goes to the plugin basics and not the flextesa page